### ISSUE TITLE
Feature-Request: Presentation mode, or associate web link to a node

### ISSUE THREAD

**Issue by agisbrec** [felixhayashi/TW5-TiddlyMap#238]
I would like to use TiddlyMap in a presentation, i.e. show the mindmap and then press on the nodes to show their content. Unfortunately, the beamers usually have small resolution, so that I have to use full screen mode. When I then press on a node, it opens a small window with the corresponding tiddler, which eats lots of screenspace. Would it be possible to show the tiddler in full-screen?

I associate a pdf file with each such node (tiddler type: 'application/pdf', field: "_canonical_uri" with a weblink). Alternatively, would it be possible to open the pdf (or any other link) directly, without the tiddler?

**Comment by tobibeer** [comment]
@agisbrec, try to double click a node.

**Comment by agisbrec** [comment]
Double clicking a node opens the corresponding tiddler in the river on the left side. I want the double click to open the pdf directly.

I managed to let pdfs appear if I set the tiddler type to 'application/pdf' and adding the field '_canonical_uri' with a weblink. Not sure how to make imported pdfs visible and whether this is a bug or I don't know the syntax. ;)

**Comment by tobibeer** [comment]
> Double clicking a node opens the corresponding tiddler in the river on the left side. I want the double click to open the pdf directly.

I thought you were talking about "presentation mode", aka fullscreen.

**Comment by agisbrec** [comment]
Yes, sorry. In fullscreen it opens a window with the name of the node and two tabs: Draft and Preview. This window eats lots of screenspace.

**Comment by tobibeer** [comment]
> This window eats lots of screenspace.

If that's your concern, then what you really ask is for a TiddlyWiki core change to have modals look differently, e.g. window-less... or even styled (via css). Unfortunately, you can't define a css class for the modal atm.

I think it's also been a loooong time that I asked @Jermolene for the ability to click outside the modal so as to close it.

**Comment by agisbrec** [comment]
OK, changing css is complicated... 
But is it possible to just redirect to the link? E.g. while editing a node tiddler, add a field 'link' with a weblink. Double clicking on the node would result in going to the linked page instead of opening the tiddler.

**Comment by felixhayashi** [comment]
Hi @agisbrec 
sorry for the late response. @tobibeer is right, you could use css styling to alter the layout of the modals. Another option would be to make use of popups in TiddlyMap (so far it only works in firefox): When you move you mouse over a node, its body is revealed as tooltip and the external links are clickable and open in another browser tab. You can try it out: go to http://tiddlymap.org?tmap-enlarged=fullscreen&tmap-view=overview and move your mouse pointer over Jeremy Ruston, then scroll down the popup till you see the links. – it's not an ideal solution, but it should work.

-felix

**Comment by agisbrec** [comment]
We already tried to modify the css. We changed the size of the new window to fullscreen, but still the headers (with tabs etc.) take too much space. I guess we will try the second option next, i.e. popup and click the link. If this still does not work, we will have to figure out how to do more complex changes to css and javascript...

Thanks!
